Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
```

Comment:
quick note, i know you didnt write this, but just a reminder that you import this css file in a higher order component, so there is no need to import it here again.